container/container.go

```diff
@@ -109,6 +109,7 @@ type Container struct {
 
 	// Fields here are specific to Windows
 	NetworkSharedContainerID string
+	SharedEndpointList       []string
```

Comment:
@cpuguy83  Can you point me out where the marshaled json is used & why do you think the windows specific fields are not needed. I am just trying to understand the use case, so that we do not miss anything for windows.
@jhowardmsft  Is it safe to make this change proposed by @cpuguy83